src/Stepper/Stepper.js

```diff
@@ -23,9 +23,9 @@ class Stepper extends Component {
      */
     activeStep: PropTypes.number,
     /**
-     * Should be two or more `<Step />` components
+     * Should be two or more `<Step />` components, but also accepts one.
```

Comment:
I'm not sure it can be one child. What about saying it's best to use with more than one `<Step />`?